cmd/tcpgoon.go

```diff
@@ -53,26 +57,65 @@ func init() {
 	runCmd.Flags().BoolVarP(&params.debug, "debug", "d", false, "Print debugging information to the standard error")
 	runCmd.Flags().IntVarP(&params.reportingInterval, "interval", "i", 1, "Interval, in seconds, between stats updates")
 	runCmd.Flags().BoolVarP(&params.assumeyes, "assume-yes", "y", false, "Force execution without asking for confirmation")
+	runCmd.Flags().BoolVarP(&params.docker, "docker", "", false, "Running against a docker image. " +
```

Comment:
re-using things like the "host" to inject the image name is a bit hacky. But that is a minor thing